third_party/gems/bundler.BUILD

```diff
@@ -0,0 +1,35 @@
+
+filegroup(
+    name = "runtime_deps",
+    srcs = [
+        "%{site_bin}/bundle",
+        "%{site_bin}/bundler",
+    ] + glob([ "%{site_ruby}/**/*.rb" ]),
+    visibility = [ "//visibility:public" ],
+)
+
+sh_binary(
+    name = "bundler",
+    data = [ ":runtime_deps", ":bundle-env" ],
+    srcs = [ "bundle.sh" ],
+    deps = [ "@bazel_tools//tools/bash/runfiles" ],
+    visibility = [ "//visibility:public" ],
+)
+
+sh_binary(
+    name = "bundle",
+    data = [ ":runtime_deps", ":bundle-env" ],
+    srcs = [ "bundle.sh" ],
+    deps = [ "@bazel_tools//tools/bash/runfiles" ],
+    visibility = [ "//visibility:public" ],
+)
```

Comment:
What's the difference between this `sh_binary` (bundle) and the one above (bundler)?

Reply:
There is no difference, this is just to preserve the expected interface to bundler, which is that both the `bundle` and `bundler` executables exist.